.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sure, I need your advice on how best to do this then.  The tutorials have to be able to be followed step by step from scratch, so if we are not going to include the project files then the tutorial steps need to be clear.  However, it is important that the tutorial can be executed easily without having to do much other than add an API key.